### PR TITLE
Remove duplicate instruction

### PR DESCRIPTION
**Pull request by tt**

### CODE REVIEW

File: platforms/ruby/Dockerfile.t [hunk 1]
```diff
@@ -25,7 +25,6 @@ RUN mkdir -p /app/heroku/node
 RUN curl -s https://s3pository.heroku.com/node/v$NODE_ENGINE/node-v$NODE_ENGINE-linux-x64.tar.gz | tar --strip-components=1 -xz -C /app/heroku/node
 ENV PATH /app/heroku/node/bin:$PATH
 
-ONBUILD WORKDIR /app/src
```
Comment (hunterloftis) [inline platforms/ruby/Dockerfile.t]: Shouldn't we leave this line, but remove `ONBUILD`? (since we just removed the `run` instruction `-w /app/src`)

Comment (tt) [inline platforms/ruby/Dockerfile.t]: It's already [here](https://github.com/tt/heroku-docker/blob/3e36cf75f0c822e937675cf0a4206248132b7c47/platforms/ruby/Dockerfile.t#L5).

Comment (hunterloftis) [inline platforms/ruby/Dockerfile.t]: That's to `/app` not `/app/src`

**Comment by hunterloftis** [comment]
Just patched it here, closing: https://github.com/heroku/heroku-docker/pull/70

**Pull request closed without merging**